components/products_list/index.js

```diff
@@ -1,22 +1,6 @@
-import { gql, useQuery } from '@apollo/client'
+import { useQuery } from '@apollo/client'
```

Comment:
We should rename the folder `products_list` to `product-list`. It's generally more common this way.